### PR TITLE
Display CLI changelog with 'doppler changelog' command

### PR DESCRIPTION
**Pull request by Piccirello**
This pulls the changelog from cli.doppler.com/changes, which is a Cloudflare Worker that pulls it from GitHub.

Screen Shot 2020-10-16 at 2 29 30 AM

Most recent changes displayed after updating:
Screen Shot 2020-10-18 at 11 57 43 AM

### CODE REVIEW

File: pkg/cmd/update.go [hunk 1]
```diff
@@ -60,6 +61,13 @@ func installCLIUpdate() {
 
 	if wasUpdated {
 		utils.Log(fmt.Sprintf("Installed CLI %s", installedVersion))
+		utils.Log("\nWhat's new:")
```
Comment (rgmvisser) [inline pkg/cmd/update.go]: We should move this inside the if statement in case we don't get a result.

**Review by rgmvisser — DISMISSED** [review]
This is great! A small addition, but really nice. Maybe we can also print something like:
`run 'doppler changelog' to see all our latest changes`?

**Review by rgmvisser — CHANGES_REQUESTED** [review]
back in your queue

**Comment by Piccirello** [comment]
Screen Shot 2020-10-27 at 6 09 25 PM

**Review by rgmvisser — APPROVED** [review]

**Pull request merged**